CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # App Center SDK for React Native Change Log
 
+## Version 2.6.2 (Under development)
+
+### App Center
+
+#### iOS
+
+* **[Fix]** Fix an error where **appcenter.podspec.json** could not be found.
```

Comment:
```suggestion
* **[Fix]** Fix an error where **appcenter.podspec.json** could not be found when using CocoaPods version 1.8.x.
```

Reply:
It's not the json file being the issue but the podspec unless I am mistaken.

Reply:
The error message looks like:
> Errno::ENOENT - No such file or directory @ rb_sysopen - /Users/joshua/pathtomy/Project/ios/Pods/Local Podspecs/appcenter.podspec.json

So this description is not wrong.